### PR TITLE
fix up navigator-specific accessibility

### PR DESCRIPTION
**Pull request by tballmsft**

### CODE REVIEW

File: navigator.ts [hunk 1]
```diff
@@ -244,42 +242,43 @@ namespace microcode {
...
             let btn = super.reportAria(b)
             if (!btn) return null
-            let status = btn.getIcon() === "note_on" ? "on" : "off"
-            let noteIndex = this.hasDelete ? this.row - 1 : this.row
-            let noteName = noteNames[noteIndex]
-
+            const status = btn.getIcon() === "note_on" ? "on" : "off"
+            const noteIndex = this.hasDelete ? this.row - 1 : this.row
+            const noteName = noteNames[noteIndex]
+            const report = `note ${noteName} in column ${
```
Comment (pelikhan) [inline navigator.ts:271]: not localizable

Comment (ramseyball) [reply navigator.ts:271]: Suggestions?

Comment (tballmsft) [reply navigator.ts]: simplified language

**Comment by pelikhan** [comment]
Still not localizable, you cannot assemble strings.

Also on/off.

**Comment by ramseyball** [comment]
Please suggest a solution. You are the expert here.

**Comment by pelikhan** [comment]
All the string mangling for screen readers should be done in custom.js and the strings should be formatted string so that they can be localized as a single string.

Maybe we can use emojis to get free translations.

**Comment by pelikhan** [comment]
The message formatting is fixing. 
- [ ] when toggling LEDs in the editor, the accessibility message is not updated.

**Comment by tballmsft** [comment]
@pelikhan  - please test my change - I can't get jekyll working.

**Comment by pelikhan** [comment]
You don't need jekull, just look at the "aria-live: ..." comments in the javascript console.


Sent from Outlook<http://aka.ms/weboutlook>

________________________________

**Comment by tballmsft** [comment]
yep. good to go!

**Pull request merged**